components/properties/src/props.rs

```diff
@@ -46,63 +48,60 @@ enum EnumeratedProperty {
 /// These are the categories required by the Unicode Bidirectional Algorithm.
 /// For the property values, see [Bidirectional Class Values](https://unicode.org/reports/tr44/#Bidi_Class_Values).
 /// For more information, see [Unicode Standard Annex #9](https://unicode.org/reports/tr41/tr41-28.html#UAX9).
+#[open_enum]
```

Comment:
nit: I would put this attribute at the very bottom since it may not be obvious from context otherwise

Reply:
Unfortunately, the order does matter here - attributes are evaluated from top to bottom. The `#[derive]` block needs to go after `#[open_enum]`, or else they will try to treat the value as a plain enum, which goes poorly (the `derive(Debug)` attempts an exhaustive match, for example). Since most of the attributes are `derive`s and the `zerovec::make_ule` must also come after `#[open_enum]`, I kept it at top for consistency.

Reply:
ah, hmmmm

The way `zerovec` handles this problem is that it exposes `#[zerovec::derive(Debug, ....)]` which is used for deriving Debug/etc on the generated type. But in zerovec's case there is an original and a generated type.